server/src/com/cloud/storage/LocalStoragePoolListener.java

```diff
@@ -18,7 +18,7 @@
 
 import javax.inject.Inject;
 
-import org.apache.log4j.Logger;
+// import org.apache.log4j.Logger;
```

Comment:
While making changes in this file, I noticed that Eclipse was pointing out that the variable s_logger was never used. Once I commented it out, Eclipse reminded me that its import statement was un-used. Instead of removing the entire line, I just commented it out in case someone later wanted to easily resume making use of this import statement.